SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -186,6 +188,7 @@ void USpatialNetDriver::OnConnectedToSpatialOS()
 
 	if (IsServer())
 	{
+		EntityPool->Init(this, TimerManager);
```

Comment:
Any reason to not do this in `CreateAndInitializeCoreClasses` ?

Reply:
`EntityPool = NewObject<UEntityPool>()` is only called in `CreateServerSpatialOSNetConnection` on the server so we only want to call `Init` on the entity pool if `IsServer()` is true, i can move it inside `CreateAndInitializeCoreClasses` and wrap inside another `IsServer` check if that's preferred?

Reply:
Yes, that sounds preferable.